clusters/build-clusters/02_cluster/README.md

```diff
@@ -0,0 +1,73 @@
+# 01-Cluster
+
+[02-Cluster](https://console-openshift-console.apps.build02.gcp.ci.openshift.org) is an OpenShift-cluster managed by DPTP-team. It is one of the clusters for running Prow job pods.
+
+## Installation
+
+The gcp project `openshift-ci-build-farm` for installation of this cluster: [gcp console](https://console.cloud.google.com/home/dashboard?project=openshift-ci-build-farm). The project is created by [DPP team](hhttps://issues.redhat.com/browse/DPP-4926).
+
+We created the public zone (base domain for installer) and the service account `ocp-cluster-installer`.
+Since this project is dedicated for build farm. We granted the `Owner` role to this account.
+
+Get the pull secret by `oc get secret -n ci cluster-secrets-gcp -o jsonpath='{.data.pull-secret}' | base64 -d`.
+
+Generate `install-config.yaml` (uploaded to BW already for reuse): 
+
+```
+./openshift-install create install-config
```

Comment:
So these steps are only needed if the `install-config.yaml` gets losed? Wouldn't it then make sense to start the paragraph off with:

* Download the `install-config.yaml` from bitwarden
* With this command you create the cluster:

And at the end have a section `Re-Generate the `install-config.yaml`?

Reply:
Done.